ansible/Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM ubuntu:16.04
```

Comment:
could probably cut this down a fair bit by using `python:2.7.14-jessie`

Reply:
I'm intending to use this as a local stand-in for building an AMI on top of a stock Ubuntu 16.04 AMI.  So I'd like this image to mirror the OS as much as reasonably possible. How about  we build a few tags of this image instead? E.g., keep the proposed Dockerfile and tag it with `ubuntu-1604` and add another Dockerfile for a smaller image (tagged with `jessie`?, `lite`?) for when folks are just looking to invoke ansible-lint or what-have-you.

Reply:
Also:

```
$ docker images | grep ansible
wpengine/ansible                                    jessie                        d5663d1bf9cc        5 minutes ago       742MB
wpengine/ansible                                    ubuntu-1604                   be19ed7b6eb3        11 minutes ago      494MB
```

and that is with whittling down the current Dockerfile's `apt install` argument list to just sudo...